framework/scripts/cluster_control.py

```diff
@@ -120,7 +120,9 @@ def calculate_seconds(start_time, end_time):
                         f"Integrity check: {node_info['status']['last_check_integrity']['date_end_master']} | " \
                         f"Integrity sync: {node_info['status']['last_sync_integrity']['date_end_master']} | " \
                         f"Agents-info: {node_info['status']['last_sync_agentinfo']['date_end_master']} | " \
-                        f"Last keep alive: {node_info['status']['last_keep_alive']}.\n"
+                        f"Last keep alive: {node_info['status']['last_keep_alive']} | " \
+                        f"Agent-groups: {node_info['status']['last_sync_agentgroup']['date_end']} | " \
+                        f"Agent-groups full: {node_info['status']['last_sync_full_agentgroup']['date_end']}.\n"
```

Comment:
I think it would be better to place these two tasks before `Last keep alive` (just a suggestion):
```suggestion
                        f"Agent-groups: {node_info['status']['last_sync_agentgroup']['date_end']} | " \
                        f"Agent-groups full: {node_info['status']['last_sync_full_agentgroup']['date_end']} | " \
                        f"Last keep alive: {node_info['status']['last_keep_alive']}.\n"
```

Reply:
Changed the format as requested:

```
# var/ossec/bin/cluster_control -i
Cluster name: wazuh

Last completed synchronization for connected nodes (1):
    worker (172.18.0.2): Integrity check: 2023-03-29T14:23:16.620492Z | Integrity sync: 2023-03-29T14:23:07.574395Z | Agents-info: n/a | Agent-groups: 2023-03-29T14:23:19.333603Z | Agent-groups full: n/a | Last keep alive: 2023-03-29T14:22:49.035152Z.
```